### ISSUE TITLE
Transforming from Video/Image to File should select caption

### ISSUE THREAD

**Issue by illusaen** [wordpress-mobile/gutenberg-mobile#3332]
**Describe the solution you'd like**
When transforming a video or image block to a file block, the caption should be autofocused after the transformation occurs.

**Describe alternatives you've considered**
Fill in the caption with the video/image file's name.

**Comment by hypest** [comment]
👋 @illusaen , I'm triaging new tickets and run into this. Should I assume this is actually a "task" for the block transform project? I will add the "Task" label and add the ticket to the Mobile Gutenberg GitHub board but we can revise if needed.

**Comment by mchowning** [comment]
I don't think this is a blocker for releasing the transform feature. Let me know if you feel differently @iamthomasbishop .

**Comment by iamthomasbishop** [comment]
> I don't think this is a blocker for releasing the transform feature. Let me know if you feel differently @iamthomasbishop .

Correct, not a blocker.
